CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Error/ErrorType.swift

```diff
@@ -139,4 +148,15 @@ import GiniBankAPILibrary
                 comment: "Outage error")
         }
     }
+
+    /**
+     Get the error analytics for the current `ErrorType`.
+
+     - Returns: An `ErrorAnalytics` object representing the analytics for the error.
```

Comment:
[Clarification]: We've discussed it already and short summary: 
 - Function is a mapping a GiniError into ErrorAnalytics
 -  it returns an error in special format for the analytics
 Answer from @ValentinaIancu-Gini : Returns: An `ErrorAnalytics` object representing the error for the analytics.

Reply:
I've added more comments and edit this one.